scripts/rotate_ci_keys.sh

```diff
@@ -51,9 +51,7 @@ echo "making kernel module key"
 GENKEY=factory-keys/x509.genkey
 if [ ! -f $GENKEY ]
 then
-	if [ ! conf/keys/x509.genkey ]
-	then
-		cat > factory-keys/x509.genkey <<EOF
+	cat > factory-keys/x509.genkey <<EOF
```

Comment:
Wouldn't this be better to keep code configurable?

`cat > $GENKEY <<EOF`

Reply:
I think is better to remove all variables and keep all keys hardcoded with the same names created at factory time

Reply:
@quaresmajose What's the point to hardcode the key name used in several lines?

Reply:
Instead of checking if the x509.genkey exist or not we should generate it always like we did for all the others. If we can do this we only we only reference the keys in two places, to generate and to use.

In any case I agree with your comment  `cat > $GENKEY <<EOF`.

Reply:
I agree with the  `cat > $GENKEY <<EOF`

and I agree that the `x509.genkey` should exist when a new factory is created. But for old factories, it will always be missing, so I think this script should still create it (at least as far as I understand)

Reply:
changed @MrCry0 @quaresmajose